src/emulator/controller.ts

```diff
@@ -272,10 +280,19 @@ export async function startAll(options: any, noUi: boolean = false): Promise<voi
     version: "unknown",
   };
   if (options.import) {
-    const importDir = path.resolve(options.import);
-    exportMetadata = JSON.parse(
-      fs.readFileSync(path.join(importDir, HubExport.METADATA_FILE_NAME), "utf8").toString()
-    ) as ExportMetadata;
+    const importFilePath = path.join(path.resolve(options.import), HubExport.METADATA_FILE_NAME);
+    if (fileExistsSync(importFilePath)) {
+      exportMetadata = JSON.parse(
+        fs
+          .readFileSync(
+            path.join(path.resolve(options.import), HubExport.METADATA_FILE_NAME),
+            "utf8"
+          )
+          .toString()
+      ) as ExportMetadata;
+    } else {
+      // could happen when an export was interrupted...
```

Comment:
Did you mean to handle this `else` case or log something?  Having it empty seems strange.

Reply:
I indeed wanted to add a warning there. Added now!